pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -660,10 +660,6 @@ func (cm *containerManagerImpl) GetResources(pod *v1.Pod, container *v1.Containe
 		if err != nil {
 			return nil, err
 		}
-		// NOTE: Passing CDI device names as annotations is a temporary solution
-		// It will be removed after all runtimes are updated
-		// to get CDI device names from the ContainerConfig.CDIDevices field
-		opts.Annotations = append(opts.Annotations, resOpts.Annotations...)
 		opts.CDIDevices = append(opts.CDIDevices, resOpts.CDIDevices...)
```

Comment:
@bart0sh we are OK completely removing support for annotations? All runtimes have supported `ContainerConfig.CDIDevice` long enough, that we don't expect this to be a problem?

Reply:
See https://github.com/kubernetes/kubernetes/issues/125210 - we agreed to make that change in 1.31.

Reply:
Correct, we can do it now.